pom.xml

```diff
@@ -93,18 +99,20 @@
 
         <!-- Logging -->
         <slf4j-api.version>1.7.25</slf4j-api.version>
-        <scala-logging.version>3.9.2</scala-logging.version>
+        <scala-logging.version>3.9.5</scala-logging.version>
         <logback-classic.version>1.2.3</logback-classic.version>
 
         <!-- Testing -->
-        <junit.jupiter.version>5.3.2</junit.jupiter.version>
+        <junit.jupiter.version>5.8.2</junit.jupiter.version>
         <junit.platform.version>1.2.0</junit.platform.version>
-        <scalatest.version>3.0.5</scalatest.version>
+        <scalatest.version>3.2.12</scalatest.version>
         <mockito.scala.version>1.4.6</mockito.scala.version>
 
         <!-- additional libraries -->
-        <json4s-jackson.version>3.6.3</json4s-jackson.version>
-        <skinny-framework.version>3.0.1</skinny-framework.version>
+        <json4s-jackson.version>4.0.5</json4s-jackson.version>
+        <skinny-framework.version>3.1.0</skinny-framework.version>
+        <maven-checkstyle-plugin.version>3.1.2</maven-checkstyle-plugin.version>
```

Comment:
if you run into checkstyle issues in the new future, upgrading to these version should solve the issue. 
(related error message: "LineLength is not allowed as a child in Checker".